quickwit/quickwit-doc-mapper/src/default_doc_mapper/mapping_tree.rs

```diff
@@ -103,6 +108,16 @@ impl LeafType {
                     Err(format!("Expected bool value, got '{}'.", json_val))
                 }
             }
+            LeafType::IpAddr(_) => {
+                if let JsonValue::String(ip_address) = json_val {
+                    let ipv6_value = IpAddr::from_str(ip_address.as_str())
+                        .map_err(|err| format!("Failed to parse ip address '{ip_address}', {err}"))?
```

Comment:
```suggestion
                        .map_err(|err| format!("Failed to parse IP address `{ip_address}`: {err}"))?
```

Reply:
Regarding the quote, using `'` makes a bit of a mixture here so I will just use ``` in the other places.